koku/koku/configurator.py

```diff
@@ -380,6 +400,26 @@ def get_kafka_topic(requestedName: str):
         """Obtain kafka topic."""
         return KafkaTopics.get(requestedName).name
 
+    @staticmethod
+    def get_kafka_username():
+        """Obtain kafka username"""
+        return LoadedConfig.kafka.brokers[0].sasl.username
+
+    @staticmethod
+    def get_kafka_password():
+        """Obtain kafka password"""
+        return LoadedConfig.kafka.brokers[0].sasl.password
+
+    @staticmethod
+    def get_kafka_sasl_mechanism():
+        """Obtain kafka sasl mechanism"""
+        return LoadedConfig.kafka.brokers[0].sasl.saslMechanism
+
+    @staticmethod
+    def get_kafka_security_protocol():
+        """Obtain kafka security protocol"""
+        return LoadedConfig.kafka.brokers[0].sasl.securityProdocol
```

Comment:
```suggestion
        return LoadedConfig.kafka.brokers[0].sasl.securityProtocol
```
It looks like this field was added in app-common-python yesterday, so we likely need to update to the latest version.

Also, `LoadedConfig.kafka.brokers[0]` contains `cacert` and `authtype` fields, I assume we need to do something with those too.

Reply:
Thanks!